arviz/plots/kdeplot.py

```diff
@@ -146,6 +147,14 @@ def plot_kde(
 
     figsize, *_, xt_labelsize, linewidth, markersize = _scale_fig_size(figsize, textsize, 1, 1)
 
+    if isinstance(values, xr.Dataset):
+        raise ValueError(
+            "Xarray dataset object detected.Use plot_posterior,plot_denstiy,plot_joint or plot_pair"
+            "instead of "
+            "plot_kde"
```

Comment:
Seems weird there's an extra line break here. Doesn't hurt anything but just curious why its there?

Reply:
The initial line was too long and pylint was failing. Added an extra line for the aesthetics :).

Reply:
Just to make sure I understand "instead of plot_kde" was added for aesthetics?

Reply:
Yes.